python/README.rst

```diff
@@ -1,7 +1,7 @@
 Vowpal Wabbit Python Wrapper
 ============================
 
-.. image:: https://badge.fury.io/py/vowpalwabbit.svg
+.. figure:: https://badge.fury.io/py/vowpalwabbit.svg
```

Comment:
Could try this? https://stackoverflow.com/a/14387603/2214524

Reply:
Yes there was an issue using a figure without a caption. I'll try this solution, and if it doesnt work I'll go back to figure and use an empty comment for the caption